src/ripple/protocol/STBase.h

```diff
@@ -157,7 +157,7 @@ class STBase
     template <class T>
     static
     STBase*
-    emplace(std::size_t n, void* buf, T&& val)
+    emplace(std::size_t n, void* buf, T&& val) noexcept
```

Comment:
`new U(std::forward<T>(val))` is not noexcept, and that one will be a little harder to fix.

Reply:
I agree. It's gotten a bit out of scope than what I originally intended. I'm out of town  this weekend, but when I get back, I'll tidy up this PR to exclude the serialized types or create a new PR entirely for the non-ST types.